<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>887</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="inputMethodHints">
   <set>Qt::ImhDigitsOnly</set>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <widget class="QStackedWidget" name="stackedWidget">
      <property name="frameShape">
       <enum>QFrame::Panel</enum>
      </property>
      <property name="currentIndex">
       <number>6</number>
      </property>
      <widget class="QWidget" name="page">
       <widget class="QWidget" name="layoutWidget">
        <property name="geometry">
         <rect>
          <x>250</x>
          <y>141</y>
          <width>361</width>
          <height>261</height>
         </rect>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <layout class="QVBoxLayout" name="verticalLayout">
           <item>
            <widget class="QLabel" name="label">
             <property name="text">
              <string>                                  ADMIN LOGIN</string>
             </property>
            </widget>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout">
             <item>
              <widget class="QLabel" name="label_2">
               <property name="text">
                <string>Username  </string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="lineEdit"/>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_3">
             <item>
              <widget class="QLabel" name="label_3">
               <property name="text">
                <string>Password  </string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="lineEdit_2">
               <property name="echoMode">
                <enum>QLineEdit::Password</enum>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <widget class="QPushButton" name="pushButton">
             <property name="text">
              <string>Login</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_4">
             <property name="text">
              <string>Dont have account?</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_2">
           <property name="text">
            <string>Sign up</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QLabel" name="label_20">
        <property name="geometry">
         <rect>
          <x>250</x>
          <y>40</y>
          <width>351</width>
          <height>101</height>
         </rect>
        </property>
        <property name="text">
         <string>DELTA IOT LIBRARY MANAGEMENT SYSTEM </string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="page_2">
       <layout class="QGridLayout" name="gridLayout">
        <item row="0" column="0">
         <widget class="QPushButton" name="pushButton_vb">
          <property name="text">
           <string>View Books</string>
          </property>
         </widget>
        </item>
        <item row="2" column="2">
         <widget class="QPushButton" name="pushButton_rb">
          <property name="text">
           <string>Return book</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QPushButton" name="pushButton_2_vu">
          <property name="text">
           <string>View User</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QPushButton" name="pushButton_6_ab">
          <property name="text">
           <string>Add Book</string>
          </property>
         </widget>
        </item>
        <item row="0" column="2">
         <widget class="QPushButton" name="pushButton_3_sb">
          <property name="text">
           <string>Issue Book </string>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QPushButton" name="pushButton_5_au">
          <property name="text">
           <string>Add User</string>
          </property>
         </widget>
        </item>
        <item row="3" column="2">
         <widget class="QPushButton" name="pushButton_logout">
          <property name="text">
           <string>Logout</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="page_3">
       <widget class="QGroupBox" name="groupBox">
        <property name="geometry">
         <rect>
          <x>230</x>
          <y>180</y>
          <width>311</width>
          <height>221</height>
         </rect>
        </property>
        <property name="title">
         <string>Add Book </string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_4">
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_3">
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_4">
             <item>
              <widget class="QLabel" name="label_6">
               <property name="text">
                <string>Book Name </string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="lineEdit_book"/>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_5">
             <item>
              <widget class="QLabel" name="label_7">
               <property name="text">
                <string>Quantity</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QComboBox" name="comboBox_addbook">
               <item>
                <property name="text">
                 <string>Select Quantity</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>1</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>2</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>3</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>4</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>5</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>6</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>7</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>8</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>9</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>10</string>
                </property>
               </item>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <widget class="QPushButton" name="pushButton_add">
             <property name="text">
              <string>Add</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="pushButton_abback">
             <property name="text">
              <string>Back</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </widget>
      <widget class="QWidget" name="page_4">
       <layout class="QVBoxLayout" name="verticalLayout_14">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_13">
          <item>
           <widget class="QLabel" name="label_5">
            <property name="text">
             <string>View Book</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QTextEdit" name="textEdit"/>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton_vbback">
            <property name="text">
             <string>Back</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="page_5">
       <widget class="QGroupBox" name="groupBox_2">
        <property name="geometry">
         <rect>
          <x>250</x>
          <y>120</y>
          <width>296</width>
          <height>172</height>
         </rect>
        </property>
        <property name="title">
         <string>Add Student</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_6">
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_5">
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_6">
             <property name="spacing">
              <number>30</number>
             </property>
             <item>
              <widget class="QLabel" name="label_8">
               <property name="text">
                <string>Student name</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="lineEdit_addstudent"/>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_7">
             <property name="spacing">
              <number>30</number>
             </property>
             <item>
              <widget class="QLabel" name="label_9">
               <property name="text">
                <string>Mo. No.</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="lineEdit_mono">
               <property name="inputMethodHints">
                <set>Qt::ImhDigitsOnly</set>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_8">
             <item>
              <widget class="QLabel" name="label_10">
               <property name="text">
                <string>E-mail ID</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="lineEdit_email"/>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_9">
             <property name="spacing">
              <number>30</number>
             </property>
             <item>
              <widget class="QPushButton" name="pushButton_addstudent">
               <property name="text">
                <string>Add</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="pushButton_asback">
               <property name="text">
                <string>Back</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </widget>
      <widget class="QWidget" name="page_6">
       <widget class="QWidget" name="layoutWidget">
        <property name="geometry">
         <rect>
          <x>250</x>
          <y>120</y>
          <width>258</width>
          <height>248</height>
         </rect>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_7">
         <item>
          <widget class="QLabel" name="label_11">
           <property name="text">
            <string>View User</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QListWidget" name="listWidget"/>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_vuback">
           <property name="text">
            <string>Back</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
      <widget class="QWidget" name="page_7">
       <widget class="QGroupBox" name="groupBox_3">
        <property name="geometry">
         <rect>
          <x>280</x>
          <y>140</y>
          <width>351</width>
          <height>281</height>
         </rect>
        </property>
        <property name="title">
         <string>Issue Book</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_9">
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_8">
           <property name="spacing">
            <number>30</number>
           </property>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_10">
             <property name="spacing">
              <number>30</number>
             </property>
             <item>
              <widget class="QLabel" name="label_12">
               <property name="text">
                <string>Book</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="lineEdit_bookname"/>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_11">
             <property name="spacing">
              <number>30</number>
             </property>
             <item>
              <widget class="QLabel" name="label_13">
               <property name="text">
                <string>Student</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QComboBox" name="comboBox_student">
               <property name="currentText">
                <string>Select Student</string>
               </property>
               <item>
                <property name="text">
                 <string>Select Student</string>
                </property>
               </item>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <widget class="QDateEdit" name="dateEdit">
             <property name="dateTime">
              <datetime>
               <hour>0</hour>
               <minute>0</minute>
               <second>0</second>
               <year>2000</year>
               <month>1</month>
               <day>1</day>
              </datetime>
             </property>
             <property name="calendarPopup">
              <bool>true</bool>
             </property>
             <property name="date">
              <date>
               <year>2000</year>
               <month>1</month>
               <day>1</day>
              </date>
             </property>
            </widget>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_12">
             <property name="spacing">
              <number>30</number>
             </property>
             <item>
              <widget class="QPushButton" name="pushButton_issue">
               <property name="text">
                <string>Issue</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="pushButton_ibback">
               <property name="text">
                <string>Back</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </widget>
      <widget class="QWidget" name="page_8">
       <widget class="QGroupBox" name="groupBox_4">
        <property name="geometry">
         <rect>
          <x>220</x>
          <y>80</y>
          <width>361</width>
          <height>331</height>
         </rect>
        </property>
        <property name="title">
         <string>Return Book</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_15">
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_10">
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_13">
             <property name="spacing">
              <number>30</number>
             </property>
             <item>
              <widget class="QLabel" name="label_14">
               <property name="text">
                <string>Student ID</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="lineEdit_SID"/>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_16">
             <property name="spacing">
              <number>30</number>
             </property>
             <item>
              <widget class="QLabel" name="label_16">
               <property name="text">
                <string>Return Date</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QDateEdit" name="dateEdit_2">
               <property name="calendarPopup">
                <bool>true</bool>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_14">
             <property name="spacing">
              <number>30</number>
             </property>
             <item>
              <widget class="QLabel" name="label_15">
               <property name="text">
                <string>Select Book</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QComboBox" name="comboBox_selectbook"/>
             </item>
            </layout>
           </item>
           <item>
            <widget class="QPushButton" name="pushButton_search">
             <property name="text">
              <string>Search</string>
             </property>
            </widget>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_21">
             <property name="spacing">
              <number>31</number>
             </property>
             <item>
              <widget class="QLabel" name="label_21">
               <property name="text">
                <string>amount</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="lineEdit_3"/>
             </item>
            </layout>
           </item>
           <item>
            <widget class="QCheckBox" name="checkBox">
             <property name="text">
              <string>Fine is collected </string>
             </property>
            </widget>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_15">
             <property name="spacing">
              <number>30</number>
             </property>
             <item>
              <widget class="QPushButton" name="pushButton_returnbook">
               <property name="text">
                <string>Return book</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="pushButton_back">
               <property name="text">
                <string>Back</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </widget>
      <widget class="QWidget" name="page_9">
       <widget class="QGroupBox" name="groupBox_5">
        <property name="geometry">
         <rect>
          <x>240</x>
          <y>110</y>
          <width>281</width>
          <height>201</height>
         </rect>
        </property>
        <property name="title">
         <string>Sign Up</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_12">
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_11">
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_17">
             <property name="spacing">
              <number>30</number>
             </property>
             <item>
              <widget class="QLabel" name="label_17">
               <property name="text">
                <string>Username</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="lineEdit_signupuser"/>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_18">
             <property name="spacing">
              <number>30</number>
             </property>
             <item>
              <widget class="QLabel" name="label_18">
               <property name="text">
                <string>Password</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="lineEdit_signuppass"/>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_19">
             <property name="spacing">
              <number>30</number>
             </property>
             <item>
              <widget class="QLabel" name="label_19">
               <property name="text">
                <string>PassWord</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="lineEdit_signuprepass"/>
             </item>
            </layout>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_20">
           <property name="spacing">
            <number>30</number>
           </property>
           <item>
            <widget class="QPushButton" name="pushButton_signup">
             <property name="text">
              <string>sign up</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="pushButton_3">
             <property name="text">
              <string>Back</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>887</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
